### PR TITLE
LTS plan post

### PR DESCRIPTION
**Pull request by mdo**
/cc @XhmikosR @bardiharborow

### CODE REVIEW

**Comment by XhmikosR** [comment]
LGTM! Only thing is to actually fill the date. I don't remember where we agreed on right now, maybe we should use the date after we released 4.3.1 and started working on v5?

File: _posts/2019-07-24-lts-plan.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Today we're formally announcing our [Long Term Support plan](https://github.com/twbs/release), a documented approach aimed at strengthening the stability and frequency of releases. As part of this initiative, each major version of Bootstrap will receive at least six months of support after it is retired, followed by six months of critical bug fixes and security updates.
+
+Starting today, Bootstrap 3 will move to end of life, and will no longer receive critical security updates.
+
+Bootstrap 4 moved to Long Term Support release on ____ and will no longer receive new features. It will continue to receive bug fixes, security updates, and documentation updates.
+
+Bootstrap 5 is under active development. You can follow our progress [on GitHub](https://github.com/twbs/bootstrap).
+
+A special thanks to [@xhmikosr](https://github.com/xhmikosr) for their tireless work in pushing us forward.
```
Comment (XhmikosR) [inline _posts/2019-07-24-lts-plan.md]: Perhaps this should be rephrased? I don't want @Johann-S or @MartijnCuppens to feel left out.

Comment (MartijnCuppens) [reply _posts/2019-07-24-lts-plan.md]: If it comes to pushing us to to get things done, credits go to you, @XhmikosR. I don't mind not getting mentioned here.

**Comment by bardiharborow** [comment]
Before merging:

- [ ] Ensure [twbs/release](https://github.com/twbs/release) includes the decided dates, as it's linked from the post.
- [ ] Regenerate the SVG timeline graphic.

File: _posts/2019-07-24-lts-plan.md (hunk 1 above)
Comment (Johann-S) [reply _posts/2019-07-24-lts-plan.md]: not really active currently, so that's ok to me too 👍

**Comment by XhmikosR** [comment]
So, are we all on the same boat here @mdo? If so we should merge this, otherwise tweak it.

**Comment by XhmikosR** [comment]
@mdo: do we ship this as is? I think we are already a little late :P

**Comment by XhmikosR** [comment]
@mdo sorry for keeping to bug you, but we really need this either shipped as is, or we need to adapt the dates.

File: _posts/2019-07-24-lts-plan.md (hunk 1 above)
Comment (Johann-S) [reply _posts/2019-07-24-lts-plan.md]: maybe we should reword a bit if just speak about @XhmikosR 🤔
`special thanks to @XhmikosR for his tireless work`

**Comment by XhmikosR** [comment]
Going to merge this since every day that passes is bad...

**Pull request merged**